src/App.tsx

```diff
@@ -1,88 +1,19 @@
-import React, { useState, useEffect, useCallback } from "react";
+import React from "react";
 import { BrowserRouter as Router } from "react-router-dom";
 import "./App.scss";
 import "antd/dist/antd.less";
-import {
-  clearSession,
-  getSession,
-  hasSession,
-  setSession,
-  updateSession,
-} from "./services/session";
```

Comment:
This service can also be deleted as it is not used anymore.

Reply:
do you mean the session.js file? If so I was planning to add the userSlice data to session storage in a different story. I was planning to use the existing session.js as a reference as a write it.